EZSwiftExtensionsTests/UIDeviceTests.swift

```diff
@@ -7,10 +7,29 @@
 //
 
 import XCTest
+@testable import EZSwiftExtensions
 
 class UIDeviceTests: XCTestCase {
 
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testGetDocumentsDirectoryPath() {
+        
+        let docPath = UIDevice.documentsDirectoryPath()
```

Comment:
This test does not really convey what exactly documentsDirectoryPath is supposed to accomplish.